tests/unit/test_wikidata.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import pytest
+from reco_utils.dataset.wikidata import (
+    search_wikidata,
+    find_wikidata_id,
+    query_entity_links,
+    read_linked_entities,
+    query_entity_description,
+)
+
+
+@pytest.fixture(scope="module")
+def q():
+    return {
+        "correct": "the lord of the rings",
+        "not_correct": "000000aaaaa",
+        "entity_id": "Q15228",
+    }
+
+
+def test_find_wikidata_id(q):
+    assert find_wikidata_id(q["correct"]) == "Q15228"
```

Comment:
Let's use the fixture for all the values we are referring to test the functions instead of hard-coding within the test. i.e. `q["entity_id"]`

Reply:
In general, it is ok to explicitly set the assert results in the tests. We are doing it in the rest of the code, [look this](https://github.com/microsoft/recommenders/blob/master/tests/unit/test_gpu_utils.py) and [this](https://github.com/microsoft/recommenders/blob/master/tests/integration/test_notebooks_pyspark.py). Other libraries like PyTorch also set the [results explicitly](https://github.com/pytorch/pytorch/blob/master/test/test_distributions.py#L1746)